server/mainPr/src/main/java/com/team23/mainPr/Domain/RentPost/Service/RentPostService.java

```diff
@@ -84,5 +84,36 @@ public RentPostResponseDto getRentPost(Integer postId) {
         return rentPostMapper.RentPostToRentPostResponseDto(result);
     }
 
+    public String postImages(List<MultipartFile> files, Integer postId) throws IOException {
+
+        if (!files.isEmpty()) {
+            for(MultipartFile file : files)
+            {
```

Comment:
```suggestion
            for (MultipartFile file : files) {
```

- 기계적인 컨벤션이 왜 자꾸 틀리는지 모르겠어요. 리포맷을 안하시는건가요?
- 괄호를 의도적으로 개행했다면 다시 고민해주세요.